base_user_role/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "User roles",
-    "version": "17.0.1.0.1",
+    "version": "17.0.1.0.2",
```

Comment:
For your information, this change is not necessary, as done by the bot when merging.

/ocabot merge patch